swarm/api/http/error.go

```diff
@@ -67,6 +75,25 @@ func initErrHandling() {
 		//assign formatted HTML to the code
 		templateMap[code] = template.Must(template.New(fmt.Sprintf("%d", code)).Parse(tname))
 	}
+
+	caseErrors = []CaseError{
```

Comment:
why dont we create that url, or redirect. it is so annoying when you are told to manually do something that the machine can do automatically

Reply:
as @homotopycolimit said, we discussed and decided error. the idea is not to introduce hash length checks (since the hashes can theoretically start with `0x`), which will strongly type us with the hashing algorithm. plus i think it would be wiser to _not_ expose different methods of implementing the same functionality to external developers through redirects. its just more stuff we'd need to maintain in the future.

Reply:
ok add the link please